spec/requests/care_plan_contacts_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+RSpec.describe 'CarePlanContacts', type: :request do
+  let(:user) { create(:user) }
+
+  describe '#create' do
+    context 'when the user is logged in' do
+      before { sign_in user }
+
+      context 'when valid params are supplied' do
+        it 'creates the care plan contact' do
+          valid_params = { name: 'Person Name', phone: '416000000' }
+
+          post care_plan_contacts_create_path, params: { care_plan_contact: valid_params }
+
+          expect(response).to be_successful
+          expect(JSON.parse(response.body)).to eq(
+            'id' => CarePlanContact.last.id,
+            'name' => CarePlanContact.last.name,
+            'phone' => CarePlanContact.last.phone
+          )
+        end
+      end
+
+      context 'when invalid params are supplied' do
+        it 'returns 422 UNPROCESSABLE ENTITY' do
+          invalid_params = { name: nil }
+
+          post care_plan_contacts_create_path, params: { care_plan_contact: invalid_params }
+
+          expect(response).to have_http_status(:unprocessable_entity)
+        end
+      end
+    end
+
+    context 'when the user is not logged in' do
+      it 'redirects to sign_in page' do
+        valid_params = { name: 'Person Name', phone: '416000000' }
+
+        post care_plan_contacts_create_path, params: { care_plan_contact: valid_params }
+
+        expect(response).to redirect_to new_user_session_path
```

Comment:
@bennpham are we keeping `redirect_to` ?

Reply:
if not, there are some other `redirect_to`s on request specs.

Reply:
@tyler-wel @camilacampos `redirect_to` seems fine when I removed `rails-controller-testing` gem so I don't have a problem with it.